tox.ini

```diff
@@ -52,8 +52,9 @@ deps=
     -rrequirements.dev.txt
     -rrequirements.txt
 commands_pre = mkdir -p coverage
-whitelist_externals = */mkdir
+allowlist_externals = */mkdir
+                      */bash
```

Comment:
Out of curiosity, do you know what the "*/" is for? I know I once used it without and that worked as well. Is it a regex match on the full path of the executable? (tox wiki seems to be down so I can't check)

Reply:
I tried just "bash" and it was still complaining and in the warning it was showing something about "/usr/bin/bash" so I just tried it like it was done for mkdir and since it didn't complain anymore I was happy to :p

Reply:
All right, guess I was doing something wrong then. Thanks!